src/docs/cookbook/testing/integration/introduction.md

```diff
@@ -7,7 +7,7 @@ prev:
   path: /docs/cookbook/plugins/picture-using-camera
 next:
   title: Performance profiling
-  path: /docs/cookbook/testing/integration/scrolling
+  path: /docs/cookbook/testing/widget/scrolling
```

Comment:
I think this should be `/docs/cookbook/testing/unit/introduction` since that's the next article in the series according to the [index](https://github.com/flutter/website/blob/main/src/docs/cookbook/index.md)

Reply:
Ah, sorry the next article should be integration/profiling

Reply:
I think this is also `/docs/cookbook/testing/integration/profiling` based on the [index](https://github.com/flutter/website/blob/main/src/docs/cookbook/index.md). Also noticed that the profiling page had both "prev" and "next" set to `/docs/cookbook/testing/unit/introduction` so added a fix for that too.